app/grandchallenge/workstation_configs/forms.py

```diff
@@ -84,6 +86,11 @@ def __init__(self, *args, read_only=False, **kwargs):
             ),
         )
 
+        self.fields["overlay_segments"].help_text += format_lazy(
+            'Refer to the <a href="{}#segmentation-masks">documentation</a> for more information',
+            reverse("documentation:detail", args=["interfaces"]),
```

Comment:
I'm only seeing this now, but does this work? Don't you need to pass the `slug` along as a `kwargs`? Also, it's better to create an environment variable for the slug so that it's configurable and can be changed if necessary. See `DOCUMENTATION_HELP_FORUM_SLUG` as example.

Reply:
"interfaces" in the slug that is passed in args.
This will create the URL: https://GRAND_CHALLENGE_DOMAIN/documentation/interfaces
locally, it will create: https://gc.localhost/documentation/interfaces
And should create the 
I will add an environment variable to enable changing it if necessary.

Reply:
Still better to use key word arguments, makes it explicit what you're passing to the resolver.

Reply:
Done